src/components/interview.ts

```diff
@@ -103,6 +103,15 @@ export const getInterviewers = async (
   return res;
 };
 
+export const getInterviewerDomainsString = async (id: string): Promise<string> => {
+  const userDomains = await getDomains(id);
+  if (userDomains.length === 0) {
```

Comment:
nit: you can simplify this with a ternary operator like so
```
return userDomains.length === 0 ? `` : getDomainsString(userDomains)
```

Reply:
Ah ok, thanks for the tip, I've never used those before, but that definitely looks shorter, thanks!